portfolio/src/main/webapp/templates/comments.html

```diff
@@ -2,27 +2,48 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Comment Section</title>
+    <title>Contact</title>
     <link rel="stylesheet" href="../styles/comments.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
   </head>
-  <body onload="getCommentList()">
-	<div id="comment-section">
-    <div id="input-area>">
-    	<h2>Comment section</h2>
-      <form id="form" autocomplete="off" >
-				<label for="username">Name:</label>
-				<input type="text-input" name="username" id="username">
-				<label for="comment">Leave a comment here:</label>
-				<input type="text-input" name="comment" id="comment">
-				<input onclick="addComment()" type="submit" value="Submit">
-    	</form> 
-		</div>
-		<div id="output-area">
-      <label for="number-comments">Maximum number of comments to display:</label> 
-			<input type="number" name="number-comments" id="number-comments" min="1" max="50" value="5">
-      <div id="comment-display"></div>
-	  </div>
+  <body onload="showCommentFeedback(); updateLoginStatusOnButton(); getCommentList()">
```

Comment:
Can these all be combined into one call?

Reply:
I have combined the first two, but getCommentList() is in a different script.